### PR TITLE
EVG-14964 Fix Known-Issue only appearing after a second 

### PR DESCRIPTION
**Pull request by malikchaya2**
[EVG-14964 ](https://jira.mongodb.org/browse/EVG-14964 )

### Description 
when clicking into a task and then using the back button, there is a second where we load bad data until the new request refreshes it. Instead, we want to wait and only show the data once the query is done loading. 

### Testing 
tested on staging to make sure the bug is gone with this fix.

### CODE REVIEW

**Review by khelif96 — CHANGES_REQUESTED** [review]
Discussed offline, But also commenting here for visibility.  This eliminates the usability of the apollo cache since we are waiting for a response before we render any data. This is especially apparent when trying to use the table pagination. 
https://user-images.githubusercontent.com/4605522/129105500-9b687a58-43f5-4bc6-944a-1f4674e07400.mov

We can try to set the [no-cache](https://www.apollographql.com/docs/react/data/queries/#no-cache) fetch policy on the task query to avoid overwriting the cache for this field.

**Review by khelif96 — CHANGES_REQUESTED** [review]

File: src/components/PatchTabs/Tasks.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
   >(GET_PATCH_TASKS, {
     variables: queryVariables,
     pollInterval,
-    fetchPolicy: "cache-and-network",
+    fetchPolicy: "no-cache",
```
[the remark this answers is not in the document]
Comment (malikchaya2) [reply src/components/PatchTabs/Tasks.tsx:60]: setting no-cache on Task.tsx doesn't fix the bug, but this does. :(

**Review by khelif96 — CHANGES_REQUESTED** [review]
Discussed offline. This ended up breaking  the cache more. :(

**Comment by malikchaya2** [comment]
> Discussed offline. This ended up breaking the cache more. :(

Yes. Hacking the cache is hard. I will try a backend approach instead.

**Pull request closed without merging**